test/extensions/quic_listeners/quiche/platform/spdy_platform_test.cc

```diff
@@ -51,6 +63,27 @@ TEST(SpdyPlatformTest, SpdyEstimateMemoryUsage) {
   EXPECT_EQ(0, spdy::SpdyEstimateMemoryUsage(s));
 }
 
+TEST(SpdyPlatformTest, SpdyLog) {
+  // SPDY_LOG macros are defined to QUIC_LOG macros, which is tested in
+  // QuicPlatformTest. Here we just make sure SPDY_LOG macros compile.
+  SPDY_LOG(INFO) << "INFO log may not show up by default.";
+  SPDY_LOG(ERROR) << "ERROR log should show up by default.";
+
+  // VLOG is only emitted if INFO is enabled and verbosity level is high enough.
+  SPDY_VLOG(1) << "VLOG(1)";
+
+  SPDY_DLOG(INFO) << "DLOG(INFO)";
+  SPDY_DLOG(ERROR) << "DLOG(ERROR)";
+
+  SPDY_DLOG_IF(ERROR, true) << "DLOG_IF(ERROR, true)";
```

Comment:
Just as an FYI; when we do the Envoy-side QUIC implementation, we should avoid using any of these macros, ideally we restrict access to them using package visibility to just the QUICHE code and PAL.

Reply:
Ack, totally agree.